packages/flutter/lib/foundation.dart

```diff
@@ -47,3 +47,16 @@ export 'src/foundation/stack_frame.dart';
 export 'src/foundation/synchronous_future.dart';
 export 'src/foundation/timeline.dart';
 export 'src/foundation/unicode.dart';
+
```

Comment:
The code below produces these warnings:

```
dartdoc:stderr:   error: generic type handled as HTML: """This is a `List<int>`."""
dartdoc:stderr:     from foundation.bar: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:55:14)
dartdoc:stderr:   error: generic type handled as HTML: """List<int> foo = [];"""
dartdoc:stderr:     from foundation.baz: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:62:14)
dartdoc:stderr:   error: generic type handled as HTML: """This is a List<int>."""
dartdoc:stderr:     from foundation.foo: (file:///b/s/w/ir/x/w/%C3%81%20flutter%20sdk/packages/flutter/lib/foundation.dart:52:14)
```
@srawlins Is this working as intended? I would have expected to only get a warning on line 52.

Reply:
Oof no that warning is definitely super broken.

Reply:
https://github.com/dart-lang/dartdoc/issues/3545